spec/controllers/home_controller_spec.rb

```diff
@@ -1,4 +1,4 @@
-require "rails_helper"
+require 'rails_helper'
```

Comment:
Working on getting this configured properly. The ruby convention is single quotes unless you need string interpolation.